zeppelin-interpreter/src/main/java/org/apache/zeppelin/interpreter/InterpreterPropertyBuilder.java

```diff
@@ -31,6 +31,16 @@ public InterpreterPropertyBuilder add(String name, String defaultValue, String d
     return this;
   }
 
+  public InterpreterPropertyBuilder add(String name,
+                                        String envName,
```

Comment:
fix it like folllowing:
`
public InterpreterPropertyBuilder add(String name, String envName, String propertyName,
        String defaultValue, String description)
`
thanks!